changelog.txt

```diff
@@ -1,5 +1,9 @@
 == Changelog ==
 
+= 9.3.2 2024-09-18 =
+
+* Update - Add hook for custom checkout validation [#51513](https://github.com/woocommerce/woocommerce/pull/51513)
```

Comment:
I believe this file is auto generated at build time, so this entry should be removed and the file left as-is.

The description here (Add hook for custom checkout validation) can be moved to the message section of the PR body (insert in the `<details>` tags).

I'd recommend changing it to mention the hook name, and its intended purpose, for example:

> Add woocommerce_checkout_validate_order_before_payment hook for custom checkout validation before payment is taken.

![image](https://github.com/user-attachments/assets/de42ff0a-4463-4e68-9a5a-c83e84a7c16b)

Reply:
Adding a comment in the PR body:

![CleanShot 2024-09-19 at 09 18 26](https://github.com/user-attachments/assets/b0ea5fd8-14e9-45b0-8a30-cd1b9e267a2d)